### PR TITLE
Add a sanity check for runs of run.py

### PR DESCRIPTION
**Pull request by gsoltis**
`validate.py` wraps a run of `run.py`, validating that `run.py` does not crash and produces the expected number of non-empty result files. 

Will need to be updated as support for additional territories are added (Guam, American Samoa, Northern Mariana Islands)

### CODE REVIEW

**Comment by gsoltis** [comment]
I shuffled a few things out of `run.py` and `CovidDatasets.py` so I could import them without triggering downloads of the datasets.

File: run.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 import datetime
 import time
 import simplejson
+from libs.build_params import r0, OUTPUT_DIR, INTERVENTIONS
```
Comment (ghop02) [inline run.py]: I would recommend just importing the module:
`from libs import build_params`

it's helpful for keeping track of where variables come from

**Comment by gsoltis** [comment]
Might be better to wait for https://github.com/covid-projections/covid-data-model/pull/37 to land, then we can probably do this more easily w/o using a subprocess

File: validate.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    os.unlink(os.path.join(result_dir, f))
+
+
+UNSUPPORTED_REGIONS = [
+  'AS',
+  'GU',
+  'MP'
+]
+
+def validate_results(result_dir: str) -> None:
```
Comment (ghop02) [inline validate.py]: This is great, I think it's worth figuring out how to run this regularly in pytest as part of the normal tests.  But I think we can do that in another PR

File: run.py (hunk 1 above)
Comment (gsoltis) [reply run.py]: Done.

File: validate.py (hunk 2 above)
Comment (gsoltis) [reply validate.py]: Sounds good, will do

File: validate.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
+import argparse
+from datetime import datetime
+import os
+import subprocess
+import sys
+
+from libs import build_params
+
+def run_run_py(datasource: str) -> None:
```
Comment (mikelehen) [inline validate.py]: Run, run!
